ma_dsn.c

```diff
@@ -284,7 +284,11 @@ my_bool MADB_DsnStoreValue(MADB_Dsn *Dsn, unsigned int DsnKeyIdx, char *Value, m
         IntValue= 0;
         for (i= 0; i < sizeof(TlsVersionBits); ++i)
         {
+#ifdef _AIX
+          if (strstr(Value, TlsVersionName[i]) != NULL)
```

Comment:
I guess it would be better to make elsewhere globally visible macro definition or strcasestr function definition for AIX, so this problem won't reoccur. But I'm gonna merge it as is now